rpcs3/Emu/SysCalls/Modules/cellGcmSys.cpp

```diff
@@ -140,7 +170,7 @@ u32 cellGcmGetNotifyDataAddress(u32 index)
  */
 vm::ptr<CellGcmReportData> _cellGcmFunc12()
 {
-	return vm::ptr<CellGcmReportData>::make(0xC0000000); // TODO
+	return vm::ptr<CellGcmReportData>::make(local_addr); // TODO
```

Comment:
local_addr is a constant, as far as I understand, is there anything wrong with this?